spec/boardchecker_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../lib/boardchecker'
+
+describe BoardChecker do
+  it 'finds a winner in horizontal positions (Game positions: 123, 456, 789)' do
+    case123 = ['x', 'x', 'x', 4, 'o', 'o', 7, 8, 9]
+    case456 = [1, 'o', 'o', 'x', 'x', 'x', 7, 8, 9]
+    case789 = [1, 'o', 'o', 4, 5, 6, 'x', 'x', 'x']
+    expect(BoardChecker.winner?(case123)).to be true
+    expect(BoardChecker.winner?(case456)).to be true
+    expect(BoardChecker.winner?(case789)).to be true
```

Comment:
Split this and other such similar setups into separate _**it**_ blocks - one for each expectation.